### PR TITLE
PMM-4764 port faq from pmm1

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+Depending on available disk space and your requirements,
+you may need to adjust data retention time.
+
+You can control data retention by passing the :option:`DATA_RETENTION`
+environment variable when :ref:`creating and running the PMM Server container
+<server-container>`.  To set environment variable, use the ``-e``
+option.  The value should be the number of hours, and requires h suffix. 
+For example, the default value of 30 days for |opt.metrics-retention| is
+``720h``, but you can decrease the retention period for |prometheus| to 8 days
+as follows::
```
Comment (rnovikovP) [inline source/faq.rst]: maybe better to have an explanation on the Variable in reference, not in the FAQ

File: source/faq.rst [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+	If a MySQL server has a lot of schemas or tables,
+	it is recommended to disable per table metrics when adding the instance:
+
+	.. prompt:: bash
+
+	   sudo pmm-admin add mysql --disable-tablestats
+
+	.. note:: Table statistics are disabled automatically
+	   if there are over 1 000 tables.
```
Comment (rnovikovP) [inline source/faq.rst]: not implemented for now

File: source/faq.rst [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+hosting provider might have default *iptables* rules on their hosts that block
+communication between |pmm-server| and |pmm-client|, resulting in *DOWN* targets
+in Prometheus. If this happens, check firewall and routing settings on the
+Docker host.
+
+Also |pmm| is able to generate a set of diagnostics data which can be examined
+and/or shared with Percona Support to solve an issue faster. You can get
+collected logs from PMM Client using the ``pmm-admin summary`` command. 
+Obtaining logs from PMM Server can be done `by specifying the
+``https://<address-of-your-pmm-server>/managed/logs.zip`` URL, or by clicking
```
Comment (rnovikovP) [inline source/faq.rst]: it can be even ``https://<address-of-your-pmm-server>/logs.zip``

let's keep it simpler

File: source/faq.rst [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+	metrics with 1 second resolution.
+
+	In case of bad network connectivity between |pmm-server| and |pmm-client| or
+	between |pmm-client| and the database server it is monitoring, scraping every
+	second may not be possible when latency is higher than 1 second.  You can change
+	the minimum resolution for metrics by passing the ``METRICS_RESOLUTION``
+	environment variable when :ref:`creating and running the PMM Server container
+	<server-container>`.  To set this environment variable, use the ``-e`` option.
+	The values can be between *1s* (default) and *5s*.  If you set a higher value,
+	|prometheus| will not start.
```
Comment (rnovikovP) [inline source/faq.rst]: see Prod. Req about 3 resolutions. there are different metrics in different resolutions and default is 5s-5s-60s for now

File: source/faq.rst [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+	The system immediately requests that you change your password. Note that, for
+	the sake of security, your password must not be trivial and pass at least the
+	dictionary check. If you do not provide your password within sixty seconds you
+	are automatically logged out. In this case use the default credentials to log in
+	again.
+
+	.. figure:: .res/graphics/png/command-line.login.3.png
+
+	   Set a new password and have full access to your system
+
```
Comment (rnovikovP) [inline source/faq.rst]: But OVF is not availible for pmm2 for now

File: source/faq.rst (hunk 1 above)
Comment (fiowro) [reply source/faq.rst]: variable reference (the options glossary) is not yet included in the pmm2 docs.

File: source/faq.rst (hunk 2 above)
Comment (fiowro) [reply source/faq.rst]: This section is hidden (tab indentation after the .. only:: showhidden directive), but not removed to simplify its further inclusion

File: source/faq.rst (hunk 4 above)
Comment (fiowro) [reply source/faq.rst]: This is a hidden section, which waits for the settings gui which definitely should be referenced here in the 1st place.

File: source/faq.rst (hunk 5 above)
Comment (fiowro) [reply source/faq.rst]: ovf is also a hidden section which waits for its time

**Pull request merged**